### ISSUE TITLE
Hierarchy keyboard navigation does not work correctly

### ISSUE THREAD

**Issue by Battlehub0x** [Battlehub0x/RTE_Docs#26]
Minor issue: when i click an item in the Hierarchy window and press up or down, it works only on the first keypress, but after that, it doesn't change object selection until i click on an other item in the Hierarchy again. (3.5.0. with Unity 2021.3.1f1)

**Comment by Battlehub0x** [comment]
Fixed here: https://github.com/Battlehub0x/RTE_AssetStore/issues/13

![image](https://user-images.githubusercontent.com/15802443/169331476-fbe6d730-4bcd-4f02-b5b3-759b660f2350.png)
